### PR TITLE
Added Transfer to MADIE option in export view

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## Description

1.  A radio button is added to the Export view.
2. This button is visible only for the measure owner and if the feature flag ( "MADIE") is set to true.
3. A service call is in place to handle the [https://jira.cms.gov/browse/MAT-3931](https://jira.cms.gov/browse/MAT-3931) upcoming story.
4. The following messages are displayed on the successful upload of measure to S3 or on failure.

image
image

## JIRA Ticket
[https://jira.cms.gov/browse/MAT-3894](MAT-3894)

## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java [hunk 1]
```diff
@@ -8,10 +8,7 @@
 import com.google.gwt.user.client.ui.Widget;
 import mat.client.buttons.CancelButton;
 import mat.client.buttons.SaveButton;
-import mat.client.shared.ErrorMessageAlert;
-import mat.client.shared.MatContext;
-import mat.client.shared.MessageAlert;
-import mat.client.shared.SpacerWidget;
+import mat.client.shared.*;
```
Comment (sb-cecilialiu) [inline src/main/java/mat/client/export/measure/ManageMeasureExportView.java:11]: Just curious, is it preferred to import * instead of individual ones?

**Review by sb-cecilialiu — COMMENTED** [review]
Looks pretty good to me!

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java (hunk 1 above)
Comment (RohitKandimalla) [reply src/main/java/mat/client/export/measure/ManageMeasureExportView.java]: I have my Intelli J settings set to max 5, I guess it did update automatically. I modified it @sb-cecilialiu

**Review by sb-cecilialiu — APPROVED** [review]

File: src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java [hunk 2]
```diff
@@ -60,12 +73,37 @@ private String buildExportURL() {
...
+				Mat.hideLoadingMessage();
+				view.getErrorMessageDisplay().createAlert(MatContext.get().getMessageDelegate().getGenericErrorMessage());
+				MatContext.get().recordTransactionEvent(null, null, null, UNHANDLED_EXCEPTION + throwable.getLocalizedMessage(), 0);
+			}
+
+			@Override
+			public void onSuccess(Boolean success) {
+				Mat.hideLoadingMessage();
+				if (success) {
+					view.displaySuccessMessage("Measure is being processed and transferred to MADIE, You will receive an email as soon as transfer is completed");
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java:91]: For user viewable content try to stick with `MADiE` (lowercase i).

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java [hunk 3]
```diff
@@ -35,6 +36,8 @@
 
 	private RadioButton allRadio = new RadioButton("format", "All");
 
+	private final RadioButton transferToMadieRadio = new RadioButton("format", "Transfer To MADIE");
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/measure/ManageMeasureExportView.java:39]: MADiE

File: src/main/java/mat/server/MeasureServiceImpl.java [hunk 4]
```diff
@@ -550,4 +550,12 @@ public boolean checkIfLibraryNameExists(String libraryName, String setId) {
         return this.getMeasureLibraryService().libraryNameExists(libraryName, setId);
     }
 
+    @Override
+    public Boolean transferMeasureToMadie(int eMeasureId) {
```
[the remark this answers is not in the document]
Comment (RohitKandimalla) [reply src/main/java/mat/server/MeasureServiceImpl.java:554]: its just the measureID

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java (hunk 3 above)
Comment (RohitKandimalla) [reply src/main/java/mat/client/export/measure/ManageMeasureExportView.java:39]: modified
